source/features/deprioritize-marketplace-link.tsx

```diff
@@ -5,9 +5,11 @@ import domLoaded from 'dom-loaded';
 import features from '../libs/features';
 
 async function init(): Promise<void> {
-	(await elementReady('.Header-link[href="/marketplace"]'))!
+	const marketPlaceLink = (await elementReady('.Header-link[href="/marketplace"]'))!;
```

Comment:
I think there’s an extra `()!` here. There’s an if that avoids the need for `!` already

Reply:
After this, it can be merged if it works. I don’t have GHE

Reply:
It works on GHE without it and on GH with it. I don't have access to an instance of GHE that does have it opted in, but assuming the markup is like GH, it'll work. If the markup isn't like GH, well it never would have worked. :)